web/server/vue-cli/src/components/Report/Report.vue

```diff
@@ -313,8 +308,7 @@ import ReportTreeKind from "@/components/Report/ReportTree/ReportTreeKind";
 import { SetCleanupPlanBtn } from "@/components/Report/CleanupPlan";
 
 import AnalysisInfoBtn from "./AnalysisInfoBtn";
-import CheckerDocumentationDialog from
-  "@/components/CheckerDocumentationDialog";
```

Comment:
Does this mean that we can just outright delete that file? Seems like only `Report.vue` used it.

Reply:
No, we cannot delete it. The Reports.vue also uses the CheckerDocumentationDialog.